### DISCUSSION TITLE
Betrayal setup

### DISCUSSION THREAD

**Discussion by mTerlier** [Lailloken/Exile-UI#429] [Q&A]
Hello,

I've done the calibration with every syndicate and every division but I can't setup the tiers 1 / 2 / 3 to each member. (I tried key 1 / 2 / 3, nothing happend).

Thus, I don't have anything showing up when I hover over a member (I need to press my 'alt' omni-key to get thing showing up).

Can you help me understand how to get the betrayal-info working ?

Thanks a lot for this amazing app.

**Comment by Lailloken** [comment]
Hmm, here's how it's set up in general (you might have done most of it already):
- enable the feature

- calibrate the image-check in the `screen-checks` section of the settings

- holding the omni-key while the syndicate board is on screen should show the overview at the top

  ![image](https://github.com/user-attachments/assets/0b57f6f8-1b9e-45b8-bdc2-7b1573cbca1c)

- hovering over a name at the top shows the rewards, hover over them and press 1-3 to add colored highlighting

  ![image](https://github.com/user-attachments/assets/54b4c3ec-a4cc-4aaf-a1c5-3faeaa01d588)

- optionally, calibrate each member and division to get reward-info while hovering over them

  ![image](https://github.com/user-attachments/assets/70310838-cfe6-49e8-a2ed-e756356dbb4d)
***
> I need to press my 'alt' omni-key to get thing showing up

So you have remapped the C-key in game?

**Reply by mTerlier** [reply]
I need to reload the app to get the thing work.

Thanks for your answer it was well documented but hard refresh need to be done I don't know why :/
